### PR TITLE
Gcm

### PR DESCRIPTION
**Pull request by trucksumo**
Remove Location functionality and Add GCM functionality

### CODE REVIEW

File: TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/LoginActivity.java [hunk 1]
```diff
@@ -258,6 +194,7 @@ public void onResponse(JSONObject response) {
                                 session.driverId = response.getString("driverId");
                                 session.name = response.getString("name");
                                 session.phone = response.getString("phone");
+                                session.gcmRegId=regid;
```
Comment (indra83) [inline TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/LoginActivity.java]: Will have to send this GCM regId to server; and use it on server side to send notifications.

Comment (trucksumo) [inline TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/LoginActivity.java]: Present Not Implemented , Just Store GCM regId

File: TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/utils/CustomLogger.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
+/**
```
Comment (indra83) [inline TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/utils/CustomLogger.java]: Remove this file if not used.

Comment (trucksumo) [inline TruckSumo_Partner/app/src/main/java/com/trucksumo/partner/utils/CustomLogger.java]: I will remove the next commit. Present using the file

**Pull request merged**